airbyte-integrations/connectors/destination-snowflake-cortex/metadata.yaml

```
@@ -41,4 +41,13 @@ data:
   tags:
     - language:python
     - cdk:python
+  connectorTestSuitesOptions:
+  - suite: unitTests
+  - suite: integrationTests
+    testSecrets:
+      - name: SECRET_DESTINATION-SNOWFLAKE-CORTEX__CREDS
+        fileName: secrets/config.json
```

```suggestion
        fileName: config.json
```

@aaronsteers the fileName should not be a path, just a file name. It's currently airbyte-ci role to determine where it will land (in the `secrets`). Happy to reconsider this.
I think this is the root cause for the error related to missing secrets you're seeing in CI.